src/v2/Apps/Partner/Components/Overview/AboutPartner.tsx

```diff
@@ -0,0 +1,57 @@
+import React from "react"
+import { Column, GridColumns, Text } from "@artsy/palette"
+import { createFragmentContainer, graphql } from "react-relay"
+import { AboutPartner_partner } from "v2/__generated__/AboutPartner_partner.graphql"
+import { RouterLink } from "v2/Artsy/Router/RouterLink"
+
+export interface AboutPartnerProps {
+  partner: AboutPartner_partner
+}
+
+export const AboutPartner: React.FC<AboutPartnerProps> = ({
+  partner: { profile, vatNumber, website },
+}) => {
+  const isEmpty = !profile?.fullBio && !vatNumber && !website
+
+  if (isEmpty) {
+    return null
+  }
+
+  return (
+    <GridColumns mb={140} gridRowGap={2}>
+      <Column span={[12, 6]}>
```

Comment:
FWIW you don't have to manually specify the `12` which is there by default if you used the form `span={6}`. Doesn't matter either way.